code/__HELPERS/unsorted.dm

```diff
@@ -2052,6 +2052,8 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			return "Fire Alarms"
 		if(CHANNEL_ASH_STORM)
 			return "Ash Storms"
+		if(CHANNEL_RADIO_TRANSMIT)
+			return "Radio Transmit"
```

Comment:
I'd probably rename this to something a bit more descriptive like... "Radio noise"? but thats just a preference. Otherwise, LGTM

Reply:
Commit pushed fixing this, changed to radio noise.